addons/medical_feedback/stringtable.xml

```diff
@@ -189,6 +189,14 @@
                 <Chinese>啟用傷者的尖叫聲</Chinese>
                 <Turkish>Yaralı birimler tarafından çığlık atmayı etkinleştir</Turkish>
             </Key>
+            <Key ID="STR_ACE_Medical_Feedback_enableHUDIndication_DisplayName">
+                <English>Enable Fractures/Tourniquets Indication</English>
```

Comment:
Agree with @veteran29 , "Indicators" sounds more natural for this feature (here and in other places) IMO.

Reply:
Oh, missed the tailings. Yeah, sounds good.